src/components/views/AgreementLine.js

```diff
@@ -123,7 +125,7 @@ const AgreementLine = ({
         paneTitle={<FormattedMessage id="ui-agreements.agreementLine" />}
         {...paneProps}
       >
-        <Info line={line} resource={resource} />
+        <Info isSuppressFromDiscoveryEnabled={handlers.isSuppressFromDiscoveryEnabled} line={line} resource={resource} />
```

Comment:
Can you break the props into multiple lines?

Reply:
Yep, have done